xml_inscriptions/transcribed/IN.Bloom.IUAM.G.63.105.36.xml

```diff
@@ -0,0 +1,128 @@
+                     <layoutDesc>
+                        <layout columns="1" writtenLines="2">
+                           <p/>
+                        </layout>
+                     </layoutDesc>
+                  </objectDesc>
+                  
+                  <!--  *************************************  <handDesc>  ********************************* -->
+                  
+                  <handDesc hands="1">
+                     <handNote ana="#impressed.inscribed"> </handNote>
+                  </handDesc>
+                  
+                  <!--  *************************************  <decoDesc>  *********************************     -->
+                  <decoDesc>
+                     <decoNote>
+                        <p>Tlow-relief figures of a seated man and two
+                           standing women facing him</p>
+                     </decoNote>
+                  </decoDesc></physDesc>
+               <history>
+                  <summary>
+                     <rs/>  <!-- content can be "fake" or blank -->
+                  </summary>
+                  
+                  <origin>
+                     <date notBefore="-0375" notAfter="-0325">ca. 350 BCE</date>
+                     <placeName ref="europe.greece.attica">Greece, Attica</placeName>
+                  </origin>
+               </history>
+            </msDesc>
+            <listBibl>
+               <bibl>
+                  <ptr type="rest-of-bibl" target="#CAT_3"/>
+                  <biblScope unit="item">399</biblScope>
+               </bibl>
+               <bibl>
+                  <ptr type="rest-of-bibl" target="#IG_II2"/>
+                  <biblScope unit="item">11840a</biblScope>
+               </bibl>
+               <bibl>
+                  <ptr type="rest-of-bibl" target="#Rudolph"/>
+                  <biblScope unit="item">38-39, no. 17</biblScope>
+               </bibl>
+            </listBibl>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <xi:include  href="http://library.brown.edu/usep_data/resources/include_taxonomies.xml">
+            <xi:fallback>
+               <p>Taxonomies for US Epigraphy controlled values</p>
+            </xi:fallback>
+         </xi:include>
+      </encodingDesc>
+      <profileDesc/>
+      <revisionDesc>
+         <change when="2010-09-29" who="CDS">Converted to September 2009 EpiDoc</change>
+         <change when="2011-05-13" who="EM">More tweaks for bib only inscriptions</change>
+         <change when="2020-02-12" who="SPapandonatou">added metadata and transcription</change>
+      </revisionDesc>
+   </teiHeader>
+   <!--facsimile element goes here--><text>
+      <body>
+         <div type="edition">
+            <ab>
+               <lb n="1"/><persName><name key="Kephisodora Κηφισοδωρα">ΚΗΦΙΣΟΔΩΡΑ</name></persName> <persName><name key="Hagno Αγνω">ΑΓΝΩ</name></persName>
```

Comment:
As a rule, if you can interpret the text, don't leave the capital letters (that is to say, don't use diplomatic transcription). Make sure to include the accentuation and write the name as you would normally expect it to appear (so, in line 1: Κηφισοδώρα rather than ΚΗΦΙΣΟΔΩΡΑ).